Code-review.md

```diff
@@ -0,0 +1,64 @@
+Code review is an important part of building quality software.  We review each
+other's code in order to:
+
+* Keep tabs on what's changing to build and maintain shared understanding of
+  our systems.
+
+* Solicit and incorporate the knowledge and experience of the team to improve
+  the code and understanding of the problem being solved.
+
+* Reduce the number of bugs or design flaws that make it into production to
+  avoid outsized impacts from only uncovering them later.
+
+Shared understanding is the most important part of code review.  It is nice to
+avoid bugs and flaws, but as it is impossible to avoid them entirely, it's
+important to focus on work that makes fixing problems _easier_.
+
+
+## Guidelines
+
+Code review interactions should primarily foster discussion and aim to produce
+rough consensus or agreement.  Suggesting changes is ok, but code reviews are
+much more than creating punch list of changes required by a gatekeeper.
```

Comment:
Had to look up what a punch list was 😄

Reply:
Ahhh, would it be better to use "check list" instead?

Reply:
Alternatively, could keep it but link to https://m-w.com/dictionary/punch+list?

Reply:
I think the specificty is nice. although checklist is more commonly used. I'm happy with a link to MW to preempt confusion :)